entrypoint.js

```diff
@@ -4,7 +4,7 @@ const fs = require('fs');
 const yaml = require('js-yaml');
 
 const { Spectral, isOpenApiv3 } = require('@stoplight/spectral');
-const { stylish } = require('@stoplight/spectral/dist/formatters/stylish');
+// const { stylish } = require('@stoplight/spectral/dist/formatters/stylish');
```

Comment:
@ktracer-isp looks like they just moved the file. See [diff](https://github.com/stoplightio/spectral/commit/66e241bd1dc63c2293ab2b0701306974a2eb37b5#diff-985d605269bfce3947f5ac487d7a5978e5262f0ffe08b20d1d2e5fd4c24cc5f0) and [`@stoplight/spectral/dist/cli/formatters/stylish`](https://github.com/stoplightio/spectral/blob/develop/src/cli/formatters/stylish.ts)